packages/integration-sdk-cli/package.json

```diff
@@ -36,6 +36,7 @@
     "@pollyjs/adapter-node-http": "^4.0.4",
     "@pollyjs/core": "^4.0.4",
     "@pollyjs/persister-fs": "^4.0.4",
+    "@types/json-diff": "^0.5.1",
```

Comment:
I know we've probably had this package in the sdk for awhile, but it looks unmaintained so it might be something worth migrating away from in the future.

Reply:
Seems that maybe this package would be preferred (https://www.npmjs.com/package/diff)

`Diff.diffJson()`

Reply:
Yeah that might work. Sadly it doesn't look like there are any that either have 0 dependencies or keep their dependencies maintained.